trace/tests/system/flask/Dockerfile

```diff
@@ -12,17 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+FROM gcr.io/google-appengine/python
+LABEL python_version=python
+RUN virtualenv --no-download /env -p python
 
-"""Environment variables for end-to-end test."""
-
-import os
-
-# MySQL settings
-MYSQL_PASSWORD = os.environ.get('MYSQL_PASSWORD')
-MYSQL_USER = os.environ.get('MYSQL_USER')
-
-# PostgreSQL settings
-POSTGRES_PASSWORD = os.environ.get('POSTGRES_PASSWORD')
-POSTGRES_USER = os.environ.get('POSTGRES_USER')
-POSTGRES_HOST = os.environ.get('POSTGRES_HOST')
-POSTGRES_DB = os.environ.get('POSTGRES_DB')
+# Set virtualenv environment variables. This is equivalent to running
+# source /env/bin/activate
+ENV VIRTUAL_ENV /env
+ENV PATH /env/bin:$PATH
+ADD . /app/
+RUN pip install -r requirements.txt
+EXPOSE 8000
```

Comment:
What's running on port 8000, and what's running on port 8080?  Is there an inconsistency with some of the other code below?

Reply:
Thanks for pointing out! Actually I'm going to remove this dockerfile since it's not used in the system test, just for running locally.